mmv1/third_party/terraform/provider/provider.go.erb

```diff
@@ -703,6 +710,42 @@ func ProviderConfigure(ctx context.Context, d *schema.ResourceData, p *schema.Pr
 			"GOOGLE_OAUTH_ACCESS_TOKEN",
 		})
 	}
+	
+	// set universe_domain based on the service account key file.
+	if config.Credentials != "" {
```

Comment:
Can we confirm we're safe to parse this with an appropriate stakeholder? Do files already contain prod / will they contain prod in the future? We want to make sure we don't introduce a future issue here if/when they get added.

Reply:
I need to double-check on this. For now the logic is to default to prod if nothing is mentioned. I don't see evidence that files will contain prod for now.

Reply:
We need to guard against future changes, so we'd want to verify that with a webpage or doc that states as much or verification with an appropriate stakeholder.

Reply:
In the DD, the new field will always be in the SA file.